src/lighthouse.rs

```diff
@@ -301,6 +301,12 @@ impl Lighthouse {
                     "Detected quorum change, bumping quorum_id to {}",
                     state.quorum_id
                 );
+            } else if participants.iter().map(|p| p.commit_failures).sum::<i64>() > 0 {
+                state.quorum_id += 1;
+                info!(
+                    "Detected commit failures, bumping quorum_id to {}",
```

Comment:
nit: maybe not necessary since the manager already logs it, but for lighthouse we could also log the participants with commit failures here?

Reply:
done